agent/bench-scripts/postprocess/generate-result

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
What do other bench-scripts do?  Could we leverage this new file for other scripts as well?